app/internal/core/middlewares/jwt.js

```diff
@@ -0,0 +1,20 @@
+import jsonwebtoken from "jsonwebtoken";
+import { constants } from 'http2';
+
+const secretKey = "secret_key_value"
```

Comment:
Extract the secretKey from .env vars
You should create a middleware for load .env params and store it in the request object